qemu/tests/in_place_upgrade.py

```diff
@@ -47,11 +47,12 @@ def run(test, params, env):
         # create an assistant user
         upgrade_test.create_ipuser(test)
         # prepare ipu test env and execute leapp tool
-        upgrade_test.run_guest_cmd(params.get("repo_leapp"))
-        upgrade_test.run_guest_cmd(params.get("ins_leapp_cmd"))
-        upgrade_test.run_guest_cmd(params.get("prepare_env"))
-        upgrade_test.run_guest_cmd(params.get("get_answer_files_source"))
-        upgrade_test.run_guest_cmd(params.get("get_answer_files"))
+        if params.get("com_install") != "yes":
```

Comment:
```suggestion
        if params.get_boolean("com_install"):
```

Reply:
Thanks ! Updated Please help to review it again !